examples/time_frequency/plot_source_space_time_frequency.py

```diff
@@ -54,7 +54,7 @@
 stcs = source_band_induced_power(epochs, inverse_operator, bands, n_cycles=2,
                                  use_fft=False, n_jobs=1)
 
-for b, stc in stcs.iteritems():
+for b, stc in stcs.items():
```

Comment:
do we have a convention that our examples are python 3 compatible but not python 2?